src/TabbedWebView.m

```diff
@@ -288,7 +288,7 @@ -(void)webView:(WebView *)sender decidePolicyForNavigationAction:(NSDictionary *
 			}
 		}
 	}
-	if (scheme == nil || [scheme isEqualToString:@""] || [scheme isEqualToString:@"http"] || [scheme isEqualToString:@"https"] || [scheme isEqualToString:@"feed"] || [scheme isEqualToString:@"file"] || [scheme isEqualToString:@"applewebdata"] || [scheme isEqualToString:@"about"])
+	if (scheme == nil || [scheme isEqualToString:@""] || [scheme isEqualToString:@"http"] || [scheme isEqualToString:@"https"] || [scheme isEqualToString:@"feed"] || [scheme isEqualToString:@"file"] || [scheme isEqualToString:@"applewebdata"] || [scheme isEqualToString:@"about"] || [scheme isEqualToString:@"data"])
```

Comment:
I think it would be clearer to do something like:
```objective-c
NSArray *acceptedSchemes = @[@"", @"http", @"https"]; // ... etc
if ([acceptedSchemes containsObject:scheme]) {
    [listener use];
}
```
What do you think?